app/src/main/res/values/gdg_themes.xml

```diff
@@ -42,11 +44,32 @@
     <item name="windowActionBar">false</item>
   </style>
   
-  <style name="Theme.GDG.Main">
+  <style name="Theme.GDG.Main" parent="Theme.GDG.WithNavDrawer">
     <item name="actionBarInsetStart">@dimen/keyline_2_minus_16dp</item>
   </style>
 
-  <style name="Theme.GDG.Special.Wtm" parent="Theme.GDG">
+  <!-- Special Event Series-->
+  <style name="Theme.GDG.Special" parent="Theme.GDG.WithNavDrawer"/>
+
+  <!-- Experts Directory -->
+  <style name="Theme.GDG.GDE" parent="Theme.GDG.WithNavDrawer"/>
```

Comment:
One advantage is that we have theme names in the Manifest that we never change. Otherwise they all become `Theme.GDG` and `Theme.GDG.WithDrawer`
The other might be that when we look at the themes.xml, we are seeing all the styles at once.